stonesoup/feeder/geo.py

```diff
@@ -91,8 +91,8 @@ class LongLatToUTMConverter(Feeder):
     """
 
     mapping = Property(
-        (float, float), default=(0, 1),
```

Comment:
There is a subtlety here: The old default of mapping as a tuple means that simple indexing into a StateVector using the tuple of the form `my_statevector[self.mapping]` fails, as it interprets the tuple as indexes over successive dimensions. 

Using the (previously necessary) pattern of `my_statevector[self.mapping, 0]` makes numpy interpret the tuple as a series of indices into the first dimension. Switching from tuple to list restores the expected behaviour in both cases. 

In essence, this only used to work because of a side-effect of the odd indexing being used before.

Reply:
I think these should be left as is, otherwise risk having mutable defaults.

Reply:
Good catch. Reverted.

I was in the mode of think tuples were not ideal for `mapping`s at that point. You have since convinced me...